docs/src/pages/components/buttons/CustomizedButtons.tsx

```diff
@@ -45,40 +43,32 @@ const BootstrapButton = experimentalStyled(Button)({
   },
 });
 
-const ColorButton = experimentalStyled(Button)<ButtonProps>(({ theme }) => ({
+const ColorButton = styled(Button)<ButtonProps>(({ theme }) => ({
   color: theme.palette.getContrastText(purple[500]),
   backgroundColor: purple[500],
   '&:hover': {
     backgroundColor: purple[700],
   },
 }));
 
-const useStyles = makeStyles((theme: Theme) => ({
-  margin: {
-    margin: theme.spacing(1),
-  },
-}));
-
 const theme = createTheme({
   palette: {
     primary: green,
   },
 });
 
 export default function CustomizedButtons() {
-  const classes = useStyles();
-
   return (
     <div>
```

Comment:
I took a path a bit different in https://github.com/mui-org/material-ui/pull/26184/files#diff-307bc6ac4306d801ec3181f826c42b63b11b0170b45e0211b1cad2d4bbcd1933. Basically, I have tried to make it easier to copy & paste. It feels like having to remove the `sx={}` prop is more daunting than having to skip the Stack when reading the source.

Reply:
Will revert the changes. I think I did this, as the `ButtonBase` has a margin, so there was issue with overriding it with just `& > *`.

Reply:
Anyway will use the Stack everywhere.

Reply:
Yeah, reverting this change 👍